packages/legacy-scripting-runner/package.json

```diff
@@ -24,10 +24,10 @@
   },
   "dependencies": {
     "async": "2.6.1",
-    "cheerio": "1.0.0-rc.2",
     "deasync": "0.1.15",
     "flat": "4.1.0",
-    "jquery-param": "1.0.1",
+    "jquery": "2.1.4",
```

Comment:
In the [legacy docs](https://platform.zapier.com/legacy/scripting#available-libraries), we indicate that `jquery` is v 1.8.3. Is this version compatible enough?

Similarly, is that old underscore version compatible with the lodash we provide?

Reply:
The doc is likely outdated. The dependency versions we use for Web Builder can be found [here](https://github.com/zapier/lambda-wrapper/blob/2e9d7523faf9bb1a12cf3ca3010b0a0568865fc8/package.json). I also double checked in Web Builder with this script:

```js
    movie_post_poll: function(bundle) {
         return [{
             'id': new Date().getTime(),
             'jqueryVersion': $.fn.jquery,
             'underscoreVersion': _.VERSION
         }];
    }
```

The result is:

![](https://zappy.zapier.com/28347462700c213412ca7041002de325.png)

So we're good ✌️